department-of-veterans-affairs/caseflow
# Your Correspondence - Assigned Tab

Resolves [Your Correspondence - Assigned Tab](https://jira.devops.va.gov/browse/APPEALS-35817)

## Acceptance Criteria

## Testing Plan
1. Login as JOLLY_POSTMAN
2. Navigate to your correspondences
3. Toggle any of the VA DOR filters. 
4. Toggle Days Waiting to make sure its sorting correctly
